sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -1578,21 +1578,89 @@ public void Cfg_ConditionalAccess()
         public void Cfg_ConditionalAccessNested()
         {
             var cfg = Build("var a = o?.method()?[10];");
-            VerifyCfg(cfg, 5);
+            VerifyCfg(cfg, 6);
 
-            var branchBlock = cfg.EntryBlock as BinaryBranchBlock;
+            var branchBlock1 = cfg.EntryBlock as BinaryBranchBlock;
             var blocks = cfg.Blocks.ToList();
-            var oNotNull = blocks[1] as BinaryBranchBlock;
-            var methodNotNull = blocks[2];
-            var assignment = blocks[3];
+            var method = blocks[1];
+            var branchBlock2 = blocks[2] as BinaryBranchBlock;
+            var o = blocks[3];
+            var assignment = blocks[4];
             var exitBlock = cfg.ExitBlock;
 
-            branchBlock.SuccessorBlocks.Should().OnlyContainInOrder(assignment, oNotNull);
-            oNotNull.SuccessorBlocks.Should().OnlyContainInOrder(assignment, methodNotNull);
-            methodNotNull.SuccessorBlocks.Should().OnlyContain(assignment);
+            branchBlock1.SuccessorBlocks.Should().OnlyContainInOrder(branchBlock2, method);
```

Comment:
I would also verify the instructions in each block, this will help the reader identify which block is which